### PR TITLE
fix(DB/quest) Leave no one behind

### PR DESCRIPTION
**Pull request by IceNet**
## CHANGES PROPOSED:
-  Fix Quest "Leave no on behind"

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3656

## TESTS PERFORMED:
- Tested on Win10 without any problems

## HOW TO TEST THE CHANGES:
- .go 5345.567383 -3130.781250 281.192322 571
- .q rem 12512 & .q a 12512
- escort the cursader to "Dr. Rogers" .go 5254.596191 -3527.240479 291.693024 571
- the quest objectives count now

## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

Notice: My 2nd PR, don't slap me pls for the weird commit, i'm learning, thanks

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1605009324701425500.sql [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+(@NPC_JOSEPHINE,1,5257.454,-3500.14,291.6933),
+(@NPC_JOSEPHINE,2,5253.089,-3516.885,291.6786),
+(@NPC_JOSEPHINE,3,5257.784,-3521.994,291.6931),
+(@NPC_JOSEPHINE,4,5256.293,-3523.494,291.6933);
+
+DELETE FROM `creature_text` WHERE `creatureid` IN (@NPC_JONATHAN,@NPC_LAMOOF,@NPC_JOSEPHINE) AND `groupid`=1;
+INSERT INTO `creature_text` (`creatureid`,`groupid`,`id`,`text`,`type`,`language`,`probability`,`emote`,`duration`,`sound`,`comment`) VALUES
+(@NPC_JONATHAN,1,0,'<sigh>... This is the end of me.',12,0,100,1,0,0,''),
+(@NPC_LAMOOF,1,0,'<sigh>... This is the end of me.',12,0,100,1,0,0,''),
+(@NPC_JOSEPHINE,1,0,'Ohhh, I... cannot go on....',12,0,100,1,0,0,'');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1605009324701425500.sql:103]: Should there be four dots at the end?

Comment (IceNet) [reply data/sql/updates/pending_db_world/rev_1605009324701425500.sql:103]: https://youtu.be/aPn9h39rwHw?t=286

In the gossip are 4 dots, so i think its correct
Searched it too, cause, 4 dots is weird.. But seems OK so

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1605009324701425500.sql:103]: It's weird.
In this video from WoD she doesn't open a gossip when he right clicks
https://youtu.be/Evx9-8J1Z8w?t=228

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1605009324701425500.sql:103]: Nvm I found this from WotLK https://www.youtube.com/watch?v=lLOpUvAxjw0 it seems to be correct in the gossip

**Comment by Si1ker** [comment]
Test. 🐱 ❌ 

/* Affected rows: 178  Found rows: 0  Warnings: 0  Duration for 18 queries: 0.016 sec. */


image

You did magic then... 'cause there's a lot of problems actually.


When you take Jonathan to the base it says his dialogue
image

then he goes to the Dr, then it disappears and it **never** counts...
image


When you pick up Lamoof, he does not go outside the room for some time (around 1+ minute)
image
image


After some moderate time, and doing some movements at long distance, the NPC starts following as you would expect since the beginning... but now there is no enough time to take him to the base so he dies in the middle of the way to the base...
image


When you pick up Josehine and take her to the Dr.. same behavior as Jonathan.. it does not count..
image


So the entire quest is bug from one way or another.

**Comment by IceNet** [comment]
Yes, its magic.

https://youtu.be/jj9x8-gHPqE

It works for me, try to dismount.

**Comment by Gierlappen** [comment]
Sry for the offtopic:

@IceNet What UI Addons are you using?
Looks good 👍🏻

**Comment by IceNet** [comment]
> Sry for the offtopic:
> 
> @IceNet What UI Addons are you using?
> Looks good 👍🏻

ElvUI/ElvUI Enhanced/ElvUI VisualProc and some other addons, PM me in discord for full list

**Comment by Si1ker** [comment]
"It works for me, try to dismount."    ?????????????????

image

**Review by Si1ker — APPROVED** [review]

**Pull request merged**